### PR TITLE
Add filter visual

### PR DESCRIPTION
**Pull request by ieschalier**
IMG_32C1F5069C6F-1

### CODE REVIEW

**Comment by ieschalier** [comment]
#4

File: navigation/MainTabNavigator.js [hunk 1]
```diff
@@ -68,6 +67,7 @@ SettingsStack.navigationOptions = {
 const appStack = createStackNavigator(
   {
     MapStack,
+    Filter: FilterScreen,
```
Comment (agoqui) [inline navigation/MainTabNavigator.js:70]: :+1:  :smiley

File: screens/Filter/Row.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import SubCategories from './SubCategories'
+
+import getIcon from './getIcon'
+
+const Row = ({ config }) => {
+  const icon = getIcon(config)
+
+  const subCategories = config.subcategories && config.subcategories[0]
+
+  const [display, toggleDisplay] = useState(false)
```
Comment (agoqui) [inline screens/Filter/Row.js:14]: so great

**Review by agoqui — APPROVED** [review]

**Review by nrcaz — APPROVED** [review]
Beau boulot :p

File: screens/Filter/Row.js (hunk 2 above)
Comment (ieschalier) [reply screens/Filter/Row.js:14]: React hook power! 🚀

**Pull request merged**